packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -149,20 +140,45 @@ const RelationModalBody = ({ id }: RelationModalBodyProps) => {
 
   const documentTitle = documentResponse.getTitle(documentLayoutResponse.edit.settings.mainField);
 
+  const getFullPageLink = (): string => {
```

Comment:
I think this is always taking us to the default locale document.

This PR might have to wait on mine since I've now added the locale to a new params object on the document meta:

https://github.com/strapi/strapi/pull/22998/files#diff-78c7e4ef41c2d71b15cc2c36898607fcbdc3ca4689b9fd9d2311da837c1d6a3e